oxenstored/poll.ml

```diff
@@ -37,7 +37,7 @@ external set_fd_limit : int -> unit = "stub_set_fd_limit"
 let get_sys_fs_nr_open () =
   try
     let ch = open_in "/proc/sys/fs/nr_open" in
-    let v = int_of_string (input_line ch) in
+    let v = Utils.int_of_string_exn (input_line ch) in
```

Comment:
Just a caveat: `int_of_string` accepts octal and hex notations - which often is not intended.